bruno/collections/Interledger/scripts/scripts.js

```diff
@@ -0,0 +1,133 @@
+const fetch = require('node-fetch')
```

Comment:
is there a way we can maybe have this file in the root folder so that we don't have the empty "scripts" folder in the collection?